waspc/ChangeLog.md

```diff
@@ -26,6 +26,7 @@ Remember to check out the [migration guide](https://wasp.sh/docs/migration-guide
 
 ### 🐞 Bug fixes
 
+- Fixed node/npm/npx/prisma command resolution on Windows, which was one of the main stoppers of supporting Wasp on native Windows. ([#3258](https://github.com/wasp-lang/wasp/pull/3258)).
```

Comment:
This should go on "unreleased". Also I'd remove or at least rephrase the last part. Now it is not clear if this change means Wasp on Windows finally works, or if it doesn't yet.